spec/integration/specialization_spec.rb

```diff
@@ -0,0 +1,128 @@
+require 'swagger_helper'
+
+describe 'Specializations API' do
```

Comment:
We have to include authorization headers for tests that can be done at a later stage